src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -230,7 +230,7 @@ def default_api_version(self):
         # to have commands show up in the hybrid profile which happens to have the latest
         # API versions
         ResourceType.MGMT_APPSERVICE: '2018-02-01',
-        ResourceType.MGMT_EVENTHUB: '2017-04-01',
+        ResourceType.MGMT_EVENTHUB: '2018-01-01-preview',
```

Comment:
For Azure Stack, it also support the new api version, right?

Reply:
yes